org-formation/800-redirects/_tasks.yaml

```diff
@@ -149,7 +149,7 @@ SchematicDevAppDnsForward:
     # ID of the api.sagebionetworks.org zone (in sageit account)
     SourceHostedZoneId: !CopyValue [!Sub '${primaryRegion}-${resourcePrefix}-sagebio-api-zone-HostedZoneId']
     # the value of the CNAME record
-    TargetHostName: !CopyValue ['schematic-dev-DockerFargateStack-LoadBalancerDNS', !Ref DnTDevAccount]
+    TargetHostName: !CopyValue ['schematic-dev-load-balancer-dns', !Ref DnTDevAccount]
```

Comment:
We would like to get staging & dev swapped over to use this new deployment. These are the 2 environments where we will be performing our final checkout of the changes and running through end-to-end testing.

Are you able to make that changing for staging?